docker_files/recipes/openfoam.core-2.4.0/build.sh

```diff
@@ -13,7 +13,7 @@ set -o errtrace
 
```

Comment:
Whenever we modify build.sh, also package version should be shifted. Please update L29 from `0.3` to `0.4`